whitelist/strict.lst

```diff
@@ -2,4 +2,14 @@
 52.48.158.112
 13.36.40.74
 geth2.golem.network
-geth.golem.network
\ No newline at end of file
+geth.golem.network
+mainnet.infura.io
+polygon-mainnet.infura.io
+polygon-rpc.com
+arb1.arbitrum.io
+api.zksync.io
+ipfs.io
+dl.dropboxusercontent.com
+docker.io
```

Comment:
```suggestion
docker.io
hub.docker.com
```

Reply:
Docker's REST api is accessed through path /api, but giving access to `hub.docker.com` probably makes sense

Reply:
Actually just checked that `registry.hub.docker.com` is an old domain, and now everything can be accessed through `docker.io`